body.tex

```diff
@@ -259,21 +266,24 @@ \subsubsection{Defining Epics}
 \label{sec:planning-epics}
 
 The plan for a six month \gls{cycle} fundamentally consists of a set of resource loaded \glspl{epic} defined in \gls{JIRA}.
-Each \gls{epic} loaded into the plan must have:
+Each \gls{epic} loaded into the plan must have this minimum set of fields filled in:
 
 \begin{itemize}
 \item
   A concrete, well defined deliverable \emph{or} be clearly described as a ``bucket'' (\S\ref{sec:bucket});
 \item
-  The \texttt{cycle} field set to the appropriate \gls{cycle};
+  The \texttt{Component} field set to the appropriate Department;
 \item
-  The \texttt{wbs} field set to the appropriate \gls{WBS} \emph{leaf} \gls{element}.
+  The \texttt{Due Date} field set to the appropriate date, which does not exceed the due date of the Milestone it is labeled to achieve.
 \item
   The \texttt{Story Points} field set to a (non-zero!) estimate of the effort required to complete the \gls{epic} in terms of \glspl{SP} (see \S\ref{sec:effort}).
-  \item
-  Tagged with the appropriate L3 or L2 milestone that the epic will contribute to schieving (or achieve in its totality).
+\item
+  The \texttt{RO Milestone ID} field set to the appropriate L3 or L2 milestone that the epic will contribute to achieving (or achieve in its totality).
+  The exact \texttt{RO Milestone ID}text can be found in the Milestone issue or Smartsheet.
 \end{itemize}
 
+The fields above are required to have values entered because they define the connection to Smartsheet where effort-tracking for the full project is done. Other fields in the epic can also be filled in as needed. 
+
 Be aware that: (Amanda: I don't think all of the following bullets are true or necessary in the Operations non-EVMS era.  Should clarify with Cathy and Wil.) 
 
 \begin{itemize}
```

Comment:
WBS (bullet 2) can be removed from this bullet list since its removed above. 
Bullet 3 - planning package can also go I believe.

Reply:
done.